### PR TITLE
feat: add etherpad service

### LINKED ISSUE #268: Add etherpad service and plugin
Related to:
- https://github.com/graasp/graasp-plugin-etherpad/issues/1

Add etherpad container to dev docker-compose, register graasp-plugin-etherpad

### PR DESCRIPTION
**Pull request by dialexo**
Closes #268 

- Adds the etherpad service devcontainer
- Fills the devcontainer postgres with the etherpad DB
- Register etherpad bridge plugin
- Update env vars

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "@fastify/cors": "7.0.0",
     "@fastify/helmet": "8.0.0",
     "@fastify/secure-session": "4.0.0",
-    "@graasp/sdk": "github:graasp/graasp-sdk",
+    "@graasp/sdk": "github:graasp/graasp-sdk#etherpad",
```
Comment (dialexo) [inline package.json:50]: - TODO: once https://github.com/graasp/graasp-sdk/pull/69 is merged
```suggestion
    "@graasp/sdk": "github:graasp/graasp-sdk",
```

File: package.json [hunk 2]
```diff
@@ -68,6 +68,7 @@
     "graasp-plugin-actions": "github:graasp/graasp-plugin-actions",
     "graasp-plugin-categories": "github:graasp/graasp-plugin-categories",
     "graasp-plugin-chatbox": "github:graasp/graasp-plugin-chatbox",
+    "graasp-plugin-etherpad": "github:graasp/graasp-plugin-etherpad#1-add-etherpad",
```
Comment (dialexo) [inline package.json:71]: - TODO: once https://github.com/graasp/graasp-plugin-etherpad/pull/3 is merged
```suggestion
    "graasp-plugin-etherpad": "github:graasp/graasp-plugin-etherpad#1-add-etherpad",
```

**Review by spaenleh — APPROVED** [review]
💎 LGTM !

**Comment by spaenleh** [comment]
@dialexo It seems the installation fails ... maybe you can check once the plugin has been merged.

**Review by spaenleh — APPROVED** [review]
💠

**Pull request merged**